src/Umbraco.Web.UI.Client/src/packages/core/workspace/contexts/tokens/submittable-tree-entity-workspace.context-token.ts

```diff
@@ -0,0 +1,13 @@
+import type { UmbWorkspaceContext } from '../../types.js';
+import type { UmbSubmittableTreeEntityWorkspaceContext } from './submittable-tree-entity-workspace-context.interface.js';
+import { UmbContextToken } from '@umbraco-cms/backoffice/context-api';
+
+export const UMB_SUBMITTABLE_TREE_ENTITY_WORKSPACE_CONTEXT = new UmbContextToken<
+	UmbWorkspaceContext,
+	UmbSubmittableTreeEntityWorkspaceContext
+>(
+	'UmbWorkspaceContext',
+	undefined,
+	(context): context is UmbSubmittableTreeEntityWorkspaceContext =>
+		'requestSubmit' in context && 'createUnderParent' in context,
```

Comment:
@nielslyngsoe @madsrasmussen I'd noticed that the Document Workspace Breadcrumbs aren't currently working on the latest `release/16.0` branch.

I've pinpointed the bug down to this line, as the property name is `_internal_createUnderParent` (not `createUnderParent`).

How should we resolve this, update this line or rename the property?